hal/halx86/apic/apictrap.S

```diff
@@ -22,23 +22,6 @@ PUBLIC ApicSpuriousService
 ApicSpuriousService:
     iretq
 
-PUBLIC HackEoi
```

Comment:
But does that mean, then, that if one were to test ROS with HAL APIC on "old" vbox, where in principle, Windows 2003 with APIC HAL does work, we would again get problems?

Reply:
w2k3 apic hal uses the same EOI mechanism. And yes, on old VBox it won't work. But who cares?

Reply:
What version(s) are what you call "old VBox"? Is VBox 4.x or VBox 5.x in that range?

Reply:
Also regarding the EOI mechanism, VGal in his last HAL PR has the mechanism Win2k3 actually uses.

Reply:
I don't know the versions. Feel free to test it. Regarding VGal's PR: It's a huge clusterfuck. All that code is falling from the sky and probably mostly straight out of IDA or rewriting code for the sake of rewriting it. It also doesn't change the EOI mechanism, because the one used here is the one used by Windows, as it is the only way to do it.